components/ILIAS/Authentication/classes/PageLanguages/PageLanguagesOverviewTable.php

```diff
@@ -18,23 +18,39 @@
 
 declare(strict_types=1);
 
-namespace ILIAS\Authentication\LogoutPage;
+namespace ILIAS\Authentication\PageLanguages;
 
 use ILIAS\Data;
 use ILIAS\UI;
 use ilArrayUtil;
 use Psr\Http\Message\ServerRequestInterface;
 use ilLanguage;
 use ilCtrlInterface;
-use ilAuthLogoutPageEditorSettings;
+use AuthPageEditorContext;
+use ilAuthPageEditorSettings;
 
-class LogoutPageLanguagesOverviewTable implements UI\Component\Table\DataRetrieval
+class PageLanguagesOverviewTable implements UI\Component\Table\DataRetrieval
 {
+    private static array $context_lng_vars =
+        [
+            AuthPageEditorContext::LOGIN->value => [
+                'pages' => 'login_pages',
+                'page' => 'login_page',
+                'param_namespace' => 'loginpage',
+                'table_action' => 'handleLoginPageActions'
+            ],
+            AuthPageEditorContext::LOGOUT->value => [
+                'pages' => 'logout_pages',
+                'page' => 'logout_page',
+                'param_namespace' => 'logoutpage',
+                'table_action' => 'handleLogoutPageActions'
+            ]
```

Comment:
And: Instead of using this array, you could add a public method to the `Enum`, e.g.

```php
public function getLanguageIdentifierFor(string $case): string
{
    // Use a match/switch and return the language key based on the $case
}
```

With this approach, we could remove the `static` variable.